packages/features/schedules/lib/use-schedule/useSchedule.ts

```diff
@@ -18,16 +18,19 @@ export type UseScheduleWithCacheArgs = {
   month?: string | null;
   timezone?: string | null;
   selectedDate?: string | null;
-  prefetchNextMonth?: boolean;
   duration?: number | null;
-  monthCount?: number | null;
   dayCount?: number | null;
   rescheduleUid?: string | null;
   isTeamEvent?: boolean;
   orgSlug?: string;
   teamMemberEmail?: string | null;
   useApiV2?: boolean;
   enabled?: boolean;
+  bookerLayout: {
+    layout: string;
+    extraDays: number;
+    columnViewExtraDays: { current: number };
+  };
```

Comment:
_bookerLayout_ is needed by _usePrefetch_ which is now called internally by useTimesForSchedule